files/sidebars/htmlsidebar.yaml

```
@@ -3,221 +3,91 @@
-        children:
-          - /Learn_web_development/Core/Structuring_content/Basic_HTML_syntax
-          - /Learn_web_development/Core/Structuring_content/Webpage_metadata
-          - /Learn_web_development/Core/Structuring_content/Headings_and_paragraphs
-          - /Learn_web_development/Core/Structuring_content/Emphasis_and_importance
-          - /Learn_web_development/Core/Structuring_content/Lists
-          - /Learn_web_development/Core/Structuring_content/Structuring_documents
-          - /Learn_web_development/Core/Structuring_content/Advanced_text_features
-          - /Learn_web_development/Core/Structuring_content/Creating_links
-          - /Learn_web_development/Core/Structuring_content/Marking_up_a_letter
-          - /Learn_web_development/Core/Structuring_content/Structuring_a_page_of_content
-          - /Learn_web_development/Core/Structuring_content/HTML_images
-          - /Learn_web_development/Core/Structuring_content/HTML_video_and_audio
-          - /Learn_web_development/Core/Structuring_content/Mozilla_splash_page
-          - /Learn_web_development/Core/Structuring_content/HTML_table_basics
-          - /Learn_web_development/Core/Structuring_content/Table_accessibility
-          - /Learn_web_development/Core/Structuring_content/Planet_data_table
-          - /Learn_web_development/Core/Structuring_content/HTML_forms
-          - /Learn_web_development/Core/Structuring_content/Debugging_HTML
-  - title: Guides
-    details: closed
-    children:
-      - link: /Web/HTML/Content_categories
-        title: Content_categories
-      - link: /Glossary/Block-level_content
-        title: Block-level_elements
-      - link: /Glossary/Inline-level_content
-        title: Inline_elements
-      - link: /Web/HTML/Quirks_Mode_and_Standards_Mode
-        title: Quirks_Mode_and_Standards_Mode
-      - link: /Web/HTML/Date_and_time_formats
-        title: Date_and_time_formats
-      - link: /Web/HTML/Constraint_validation
-        title: Constraint_validation
-      - link: /Web/HTML/Microdata
-        title: Microdata
-      - link: /Web/HTML/microformats
-        title: Microformats
-      - link: /Web/HTML/Viewport_meta_tag
-        title: Viewport_meta_tag
-      - link: /Web/HTML/CORS_enabled_image
-        title: Allowing_cross-origin_images_and_canvas
+  - type: section
+    link: /Web/HTML/Guides
+    title: Guides
+  - type: listSubPages
+    path: /Web/HTML/Guides
+  - type: section
+    link: /Web/HTML/How_to
+  - type: listSubPages
+    path: /Web/HTML/How_to
   - type: section
     link: /Web/HTML/Reference
     title: Reference
   - type: listSubPages
-    path: /Web/HTML/Element
+    path: /Web/HTML/Reference/Elements
     title: HTML_Elements
     details: closed
     code: true
   - type: listSubPages
-    path: /Web/HTML/Global_attributes
-    title: Global_attributes
+    path: /Web/HTML/Reference/Elements/input
+    title: Input_types
     details: closed
     code: true
   - type: listSubPages
```

Can do in a follow-up if you like, but leaving this here if we want to include at some point

```suggestion
  - type: listSubPages
    path: /Web/HTML/Reference/Elements/script/type
    link: /Web/HTML/Reference/Elements/script/type
    title: Script_types
    details: closed
    code: true
  - /Web/HTML/Reference/Elements/meta/name
  - type: listSubPages
```

---

Sounds good!